### DISCUSSION TITLE
Welcome to qBoSlider Discussions!

### DISCUSSION THREAD

**Discussion by iAlexeyProkhorov** [iAlexeyProkhorov/qBoSlider#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by iAlexeyProkhorov** [comment]
Hi👋❕
Few days ago I start qBoSlider update to next version. This version will get only one new feature - slide search function in admin panel.
This should make plugin more user friendly for administrators.

Search will work for next fields:

- **Name**. Slide search name;
- **Widget zone**. You can find slide by related widget zone;
- **Start date**. Filter slides by start date;
- **End date**. Filter slides by end date;
- **Published**. Filter slides by publication flag.

**Comment by RichJeffries** [comment]
How do I stop my slides from being cropped?
I have configured a max width of 800px (the width of my slides), but both the left and right appear to be cropped?

![Screenshot 2023-09-18 101006](https://github.com/iAlexeyProkhorov/qBoSlider/assets/18453931/981cb3c4-58e4-4bcf-9b51-543812774aa1)
![Banner_Welcome](https://github.com/iAlexeyProkhorov/qBoSlider/assets/18453931/ddcc95fc-7083-4b14-94b7-dbf7532e5adc)

**Comment by iAlexeyProkhorov** [comment]
Hello again!
Thank you for your message. Now I can try fix this bug).
Slide captions located on picture or created like HTML content?
________________________________

**Reply by RichJeffries** [reply]
Hello!

The caption was added using the textbox - I thought it might be needed to force the image over.
Ideally, I don't want any caption, just the image for this particular setup.

Cheers
Rich

**Reply by iAlexeyProkhorov** [reply]
Hi! I'll try to fix this. Can you send me HTML for your slide?

**Comment by RichJeffries** [comment]
Hi Alexey,
There was no html - only the image as sent already.
